.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
### kindly you can use a full version of a gitIgnore file for node.js dev just like this file example: [.gitignore file](https://github.com/khitermedachraf/To-Do-list/blob/main/.gitignore)
**PS: You can generate it automatically when you are creating your repository.**